tensorflow/tools/docker/Dockerfile.devel-gpu-cuda9-cudnn7

```diff
@@ -79,23 +80,43 @@ RUN git clone https://github.com/tensorflow/tensorflow.git && \
     git checkout ${TF_BRANCH}
 WORKDIR /tensorflow
 
-# Configure the build for our CUDA configuration.
-ENV CI_BUILD_PYTHON python
-ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
-ENV TF_NEED_CUDA 1
-ENV TF_CUDA_COMPUTE_CAPABILITIES 3.0,3.5,5.2,6.0,6.1
-ENV TF_CUDA_VERSION 9.0
-ENV TF_CUDNN_VERSION 7
-RUN ./configure
-
-RUN LD_LIBRARY_PATH=/usr/local/cuda/lib64/stubs:${LD_LIBRARY_PATH} \
-    bazel build -c opt --config=cuda --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" \
-        --jobs=${TF_AVAILABLE_CPUS} \
-        tensorflow/tools/pip_package:build_pip_package && \
-    mkdir -p /pip_pkg && \
+# Tell TensorFlow how to configure the build, in detail.
+ENV CI_BUILD_PYTHON=python \
+    LD_LIBRARY_PATH=/usr/local/cuda/extras/CUPTI/lib64:${LD_LIBRARY_PATH} \
+    CUDNN_INSTALL_PATH=/usr/lib/x86_64-linux-gnu \
+    PYTHON_BIN_PATH=/usr/bin/python \
+    PYTHON_LIB_PATH=/usr/local/lib/python2.7/dist-packages \
+    TF_NEED_CUDA=1 \
+    TF_CUDA_VERSION=9.0 \
+    TF_CUDA_COMPUTE_CAPABILITIES=3.0,3.5,5.2,6.0,6.1,7.0 \
+    TF_CUDNN_VERSION=7 \
+    TF_NEED_JEMALLOC=1 \
+    TF_NEED_GCP=0 \
```

Comment:
No need for all of these variables. Everything starting with TF_NEED_JEMALLOC and below here are being deprecated.

Reply:
Agree.